docs/rules/no-restricted-properties.md

```diff
@@ -0,0 +1,73 @@
+# Disallow certain object properties (no-restricted-properties)
+
+Certain properties on objects may be disallowed in a codebase. This is useful for deprecating an API or restricting usage of a module's methods. This rule looks for accessing a given property key on a given object name, either when reading the property's value or invoking it as a function. You may specify an optional message to indicate an alternative API or a reason for the restriction.
+
+## Rule Details
+
+This rule is aimed at disallowing certain object properties from your code.
+
+### Options
+
+This rule takes a list of objects, where the object name and property names are specified:
+
+```json
+{
+    "rules": {
+        "no-restricted-properties": [2, [{
```

Comment:
We recently replaced number 2 with string "error" for error level in json and js examples